client/src/components/web-extension/Common/Account/AccountManagerModal.jsx

```diff
@@ -47,6 +52,16 @@ export const AccountManagerModal = ({ isOpen, onClose, isExistUser, ...restProps
 		});
 	};
 
+	const onViewPrivateKey = useCallback(
+		async (password) => {
+			const privateKey = await getPrivateKey(password);
+			navigate('/viewPrivateKey', {
```

Comment:
I reused the `PasswordForm` which already handled the failed password there. So this is the callback after the password was verified successfully. 
We need password here because we dont want to send sensitive data from SW to client without password.

Reply:
![image](https://user-images.githubusercontent.com/6711653/201965015-05fa41ba-e62c-46a5-ba9a-1571673ee0ff.png)
I fixed the issue